package.json

```
@@ -56,7 +59,7 @@
     "p-defer": "^3.0.0",
     "pre-commit": "^1.2.2",
     "prettier": "^2.0.1",
-    "webpack": "^4.42.0"
+    "webpack": "^5.1.3"
```

```suggestion
    "webpack": "^4.42.0"
```

crap, we need to revert this.